id_minter/src/test/scala/uk/ac/wellcome/platform/idminter/utils/IdentifiableTest.scala

```diff
@@ -16,4 +16,15 @@ class IdentifiableTest extends FunSpec with PropertyChecks with Matchers {
     }
   }
 
+  // We have a rule that identifiers can't start with a number, as this
+  // makes them awkward to use as a string in certain contexts.
+  it("should never generate an identifier that starts with a number") {
```

Comment:
👍 for comment in only one place. I prefer the comment alongside the code, so that it’s all in one place, rather than split across multiple tests – although I don’t think it makes a lot of difference.